request/migrations/0005_alter_request_user.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
```

Comment:
I think that you can delete this line:
```python
import django.db.models.deletion
```
 and use `on_delete=models.SET_NULL` instead of `on_delete=django.db.models.deletion.SET_NULL`.